corehq/motech/fhir/models.py

```diff
@@ -115,6 +115,12 @@ class FHIRResourceProperty(models.Model):
     # do what you need.
     value_source_config = JSONField(null=True, blank=True, default=None)
```

Comment:
Given the comment here, should `jsonpath` be directly editable in that admin view as well, or in those cases, would the data dictionary be used instead?

Reply:
Yes, exactly. The Data Dictionary lets you set `jsonpath` with a simpler UI that this one, and should cater for at least 90% of use cases.

I tried showing the other fields in the "FHIRResourcePropertyInline" model, but it looked pretty ugly, and I figured I shouldn't spend too much time trying to make it look nicer. I'm hoping we won't need to use the admin UI a lot.